test/test_markdown_report.py

```diff
@@ -145,6 +143,43 @@ def testSortRefactoringCandidatesTableBySeverity(self):
 
         self.assertEqual(table.strip(), inspect.cleandoc(expected).strip())
 
+    def testLimitRefactoringCandidatesTableWhenThereAreTooMany(self):
+        findings = [self.toRefactoringCandidate(f"aap-{i}", "introduced", "UNIT_SIZE", "HIGH") for i in range(1, 100)]
+
+        report = MarkdownReport()
+        table = report.renderRefactoringCandidatesTable(findings)
+
+        expected = """
+            | Risk | System property | Location |
+            |------|-----------------|----------|
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-1 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-2 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-3 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-4 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-5 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-6 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-7 |
+            | 🟠 | **Unit Size**<br />(Introduced) | aap-8 |
+            | ⚫️ | | + 91 more |
```

Comment:
I'm reading that `range` in Python is exclusive (we're generating 99 refactoring candidates, not 100). In that case, this is correct, but it is a bit weird. Probably nothing we can do about it.

Reply:
It's because I start at 1 instead of 0. So the `range(1, 100)` will include everything between 1 and 99, which is indeed 99 elements.